### DISCUSSION TITLE
Changes to i3 config file won't apply

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5622] [Q&A] [answered]
running `i3 --more-version` points to ``/home/alex/.config/i3/config`` which is the file I've been editing on. The file looks like this 

```
set $mod Mod4

# Use pactl to adjust volume in PulseAudio.
bindsym XF86AudioRaiseVolume exec --no-startup-id pactl set-sink-volume @DEFAULT_SINK@ +10% && $refresh_i3status
bindsym XF86AudioLowerVolume exec --no-startup-id pactl set-sink-volume @DEFAULT_SINK@ -10% && $refresh_i3status
bindsym XF86AudioMute exec --no-startup-id pactl set-sink-mute @DEFAULT_SINK@ toggle && $refresh_i3status
bindsym XF86AudioMicMute exec --no-startup-id pactl set-source-mute @DEFAULT_SOURCE@ toggle && $refresh_i3status

# Use Mouse+$mod to drag floating windows to their wanted position
floating_modifier $mod

# kill focused window
bindsym $mod+q kill

# start browser
bindsym $mod+b exec "google-chrome-stable -force-dark-mode"

# take a screenshot
bindsym --release Print exec --no-startup-id "gnome-screenshot -a -c"

# start alacritty terminal
bindsym $mod+Return exec i3-sensible-terminal

# enter fullscreen mode for the focused container
bindsym $mod+f fullscreen toggle

# reload the configuration file
bindsym $mod+Shift+c reload

gaps inner 10px

default_border pixel 0
```
meanwhile my screen looks like this (titlebars should be gone, there should be gaps, status bar shouldn't be there) 

![Screenshot from 2023-08-12 16-24-59](https://github.com/i3/i3/assets/113855995/a8b623b1-3ca1-4150-8d37-c1263651a4a1)

also, my keybinds work but whenever I add anything or change anything it doesn't get applied

**Comment by orestisfl** [comment]
You need to reload your config file each time you update it with
$mod+Shift+c, did you do that?

**Reply by ghost** [reply]
I did each change, even after rebooting my computer the problem persists

**Reply by orestisfl** [reply]
Then, can you please follow the instructions [here](https://i3wm.org/docs/debugging.html) to provide a log file? Prefer doing it for a new session: start i3 with `i3 --shmlog-size=26214400`, verify settings not applied, upload debug log: `DISPLAY=:0 i3-dump-log | bzip2 -c | curl --data-binary @- https://logs.i3wm.org`

**Comment by orestisfl** [comment, marked as answer]
Marking as solved
